packages/framework/src/Component/AddSentryContextSubscriber.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component;
+
+use App\Environment;
+use RedisException;
+use Sentry\State\Scope;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Domain\Exception\NoDomainSelectedException;
+use Shopsys\FrameworkBundle\Component\Localization\DisplayTimeZoneProviderInterface;
+use Shopsys\FrameworkBundle\Component\Maintenance\MaintenanceModeSubscriber;
+use Shopsys\FrameworkBundle\Component\Redis\RedisClientFacade;
+use Shopsys\FrameworkBundle\Component\Redis\RedisFacade;
+use Shopsys\FrameworkBundle\ShopsysFrameworkBundle;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+use Symfony\Component\HttpKernel\KernelEvents;
+use function Sentry\configureScope;
+
+class AddSentryContextSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Redis\RedisClientFacade $redisClientFacade
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     * @param \Shopsys\FrameworkBundle\Component\Localization\DisplayTimeZoneProviderInterface $displayTimeZoneProvider
+     * @param \Shopsys\FrameworkBundle\Component\Redis\RedisFacade $redisFacade
+     */
+    public function __construct(
+        protected readonly RedisClientFacade $redisClientFacade,
+        protected readonly Domain $domain,
+        protected readonly DisplayTimeZoneProviderInterface $displayTimeZoneProvider,
+        protected readonly RedisFacade $redisFacade,
+    ) {
+    }
+
+    public function setSentryContext(): void
+    {
+        $context = [
+            'environment' => class_exists('App\Environment') ? Environment::getEnvironment() : '',
```

Comment:
is this ever going to be anything else from `PRODUCTION`? In the other environment, there should not be sentry reporting at all, or? https://github.com/shopsys/shopsys/blob/16.0/project-base/app/config/packages/sentry.yaml#L1

Reply:
What if you will turn it on ? :D Like I would love to be able to easily enable Sentry on localhost :D